### PR TITLE
Adds run_job

### PR DESCRIPTION
**Pull request by benthomasson**
Experimenting with ideas for AAP-5082

### CODE REVIEW

File: ansible_events_ui/api.py [hunk 1]
```diff
@@ -507,6 +509,52 @@ async def list_job_instances(db: AsyncSession = Depends(get_db_session)):
     return result.all()
 
 
+@router.post("/api/job_instance/")
+async def create_job_instance(
+    j: JobInstance, db: AsyncSession = Depends(get_db_session)
```
Comment (cutwater) [inline ansible_events_ui/api.py:514]: Please use more descriptive variables names. Single character variables should be avoided unless they're used as a loop index or their scope is narrow.

`j` is typically used as a loop index.

**Review by cutwater — CHANGES_REQUESTED** [review]

File: ansible_events_ui/api.py [hunk 2]
```diff
@@ -507,6 +509,52 @@ async def list_job_instances(db: AsyncSession = Depends(get_db_session)):
...
+    job_uuid = str(uuid.uuid4())
+
+    query = insert(job_instances).values(uuid=job_uuid)
+    result = await db.execute(query)
+    await db.commit()
+    (job_instance_id,) = result.inserted_primary_key
+
+    event_log = asyncio.Queue()
+
+    task = asyncio.create_task(
```
Comment (cutwater) [inline ansible_events_ui/api.py:535]: I think we should document somewhere (if it hasn't been done already) that the tasking system implementation we use brings a limitation on number of worker processes that are used to run the application. The application is limited to only a single process mode and will have unexpected side effects if run in multi process mode.

**Comment by cutwater** [comment]
@benthomasson 

I think we should document somewhere (if it hasn't been done already) that the tasking system implementation we use brings a limitation on number of worker processes that are used to run the application. The application is limited to only a single process mode and will have unexpected side effects if run in multi process mode.

**Comment by benthomasson** [comment]
@cutwater  At this point, this server is only meant to work in single process mode.   In the future, multiple process mode can be supported if it is needed to meet some performance target that single process mode fails to support.

**Comment by benthomasson** [comment]
@lgalis please review the UI portions of this PR.

**Review by lgalis — APPROVED** [review]

**Pull request merged**